RiveSharp.Views.UWP/StateMachineInput.cs

```diff
@@ -1,62 +1,75 @@
 ﻿using System;
 using Windows.UI.Xaml;
+using Windows.UI.Xaml.Markup;
 
 namespace RiveSharp.Views
 {
-    // Wraps a custom, named state machine input value. This element is not visible and does not
-    // participate in layout, but it is designed to live in the XAML tree and apply its Value to its
-    // parent "RivePlayer" object's Scene.
-    public abstract class StateMachineInput : FrameworkElement
+    // StateMachineInput elements in XAML should be nested inside <StateMachineInputs>:
+    //
+    //   <rive:RivePlayer Source="...">
+    //     <rive:StateMachineInputs>
+    //       <rive:Bool ... />
+    //     </rive:StateMachineInputs>
+    //   </rive:RivePlayer>
+    //
+    public class StateMachineInputs : DependencyObjectCollection { }
+
+    // This base class wraps a custom, named state machine input value.
+    public abstract class StateMachineInput : DependencyObject
     {
-        public StateMachineInput()
-        {
-            this.Visibility = Visibility.Collapsed;
-            this.Loaded += (object s, RoutedEventArgs e) => Apply();
-        }
-
         private string mInputName;
         public string InputName
```

Comment:
Is there a term you use in Rive for these identifiers/labels of conditions/inputs/triggers?

Maybe just using `Target` would work well to kind of match how [`Setter`s](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.setter.target) work from the XAML perspective. @niels9001 thoughts?

Reply:
The runtime calls them "name", which obviously conflicts with XAML.

I think Target sounds good. Let's see what others think.